common/sgx/tcbinfo.h

```diff
@@ -15,6 +15,16 @@
 OE_EXTERNC_BEGIN
 
 #define OE_TCB_LEVEL_STATUS_UNKNOWN (0)
+// Info should be of type oe_parsed_tcb_info_t, field should be one of common
+// fields among oe_parsed_tcb_info_v2_t and oe_parsed_tcb_info_v3_t Valid fields
+// are: version, issue_date, next_update, fmspc, pceid, tcb_type,
+// tcb_evaluation_data_number If version is not 3, defaults to return v2 (which
+// also handles v1)
+#define OE_TCB_INFO_GET(info, field) \
+    (info.version == 3 ? info.tcb_info_v3.field : info.tcb_info_v2.field)
+// Info should be of type oe_parsed_tcb_info_t*
```

Comment:
do we support v1?

Reply:
Yes. tcb_info_v2 can handle both V2 and V1